app/src/lib/stores/app-store.ts

```diff
@@ -2208,7 +2207,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
   }
 
   /** This shouldn't be called directly. See `Dispatcher`. */
-  public async _repositoryWithRefreshedGitHubRepository(
+  private async _repositoryWithRefreshedGitHubRepository(
```

Comment:
I think the `_` prefix is only used for public functions that shouldn't be called. Now that this is private, shouldn't the prefix be removed?